### PR TITLE
Feature #101

### PR DESCRIPTION
**Pull request by jonstuebe**
I'm not sure exactly how to go about this. But I created a feature request on dev.fuelphp.com and went ahead and started the class myself. If this isn't the proper way to go about putting a request for the file, then let me know.

### CODE REVIEW

File: classes/typography.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Fuel\Core;
+
+class Typography {
+
+	public static function auto($str)
+	{
+		View::$auto_encode = false;
+
```
Comment (jschreuder) [inline classes/typography.php]: I can be quite clear about this: a class that auto-disables output encoding will never go into the core. This instruction should go.

**Comment by jschreuder** [comment]
Looking it over I think it's a good idea in principle, it's a bit feature light currently though.

Also I think you could do without a special html_entities method, you could first htmlentities it and only after that convert newlines to linebreaks and paragraphs. Would be far better practice than partially duplicating build in behavior.

**Comment by jonstuebe** [comment]
I've went ahead and adjusted the class. I know it's a little light on the features. But I think it's something to start with and we can add more later.

**Comment by dhrrgn** [comment]
I don't like this here.  If anything it should go in the Html or the Str class.

**Comment by dhrrgn** [comment]
You obviously don't care about this too much, haven't responded in over a week.  Closing.  Read my comments above if you want this in.

**Pull request closed without merging**